packages/quantic/force-app/main/default/lwc/quanticDocumentSuggestion/e2e/quanticDocumentSuggestion.e2e.ts

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable no-await-in-loop */
+import {test, expect} from './fixture';
+
+const fakeTrackingId = 'tracking_id_123';
+
+['legacy', 'next'].forEach((mode) => {
+  test.describe(`quantic document suggestion ${mode} analytics`, () => {
+    let analyticsMode = 'legacy';
+    test.beforeEach(async ({documentSuggestion, page}) => {
+      analyticsMode = mode;
+      if (analyticsMode === 'next') {
+        await page.context().addCookies([
+          {
+            name: 'Coveo-Pendragon',
+            value: fakeTrackingId,
+            domain: '.my.site.com',
+            path: '/',
+          },
+          {
+            name: 'LSKey-c$Coveo-Pendragon',
+            value: fakeTrackingId,
+            domain: '.my.site.com',
+            path: '/',
+          },
+        ]);
+      }
+
+      await expect(documentSuggestion.noSuggestionsMessage).toBeVisible();
+    });
+
+    test('should render the component and all parts', async ({
```

Comment:
why are we adding this test here? It seems to already be tested in the unit tests. I would remove it as we agreed to put the rendering tests in the unit tests, unless you deem its useful for some specific reason to test it here.

Reply:
true, I kinda added this one to test my setup first 🙃